### ISSUE TITLE
computer interface - this shouldn't return true

### ISSUE THREAD

**Issue by Nex4rius** [gcewing/SGCraft#46]
<img src="http://imgur.com/SZcqf42.png" />
<img src="http://imgur.com/UtCVjNg.png" />

Workaround:
<code>if component.isAvailable("stargate") then</code>
<code>  sg = component.getPrimary("stargate")</code>
<code>  if sg.energyToDial(sg.localAddress()) then</code>
<code>    return true</code>
<code>  end</code>
<code>end</code>

**Comment by gcewing** [comment]
I don't think this is incorrect; component.isAvailable is just testing whether there is an SG interface unit attached. Whether there is a stargate attached to the interface is another matter.
